lib/index.js

```diff
@@ -105,6 +105,10 @@ const downloadFromInfoCallback = (stream, info, options) => {
         Range: `bytes=${options.range.start || '0'}-${options.range.end || ''}`,
       });
     }
+    if (format.url === undefined) {
```

Comment:
internal/validators.js:120:11/TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received undefined